zipline/algorithm.py

```diff
@@ -325,16 +325,10 @@ def __init__(self, *args, **kwargs):
             cleanup=clear_dataframe_indexer_caches
         )
 
-        if 'blotter' in kwargs and 'blotter_class' in kwargs:
-            raise RuntimeError('both blotter and blotter_class are defined')
         self.blotter = kwargs.pop('blotter', None)
-        self.cancel_policy = kwargs.pop('cancel_policy', NeverCancel())
+        cancel_policy = kwargs.pop('cancel_policy', NeverCancel())
```

Comment:
Would we want to make this conditional on `not self.blotter` below?

Reply:
I did this originally, but doing so breaks our consumers downstream because we expect this kwarg to always get popped. I've made substantial changes to the TradingAlgorithm constructor in https://github.com/quantopian/zipline/pull/2192, so my inclination is to punt on cleaning this up until after that refactor langds.

Reply:
Got it. Alrighty.